app/src/main/java/com/arygm/quickfix/MainActivity.kt

```diff
@@ -237,8 +237,6 @@ fun SearchNavHost(
       startDestination = Screen.SEARCH,
       modifier = Modifier.padding(innerPadding),
   ) {
-    composable(Screen.SEARCH) {
-      QuickFixFinderScreen(navigationActions, navigationActionsRoot, isUser)
-    }
+    composable(Screen.SEARCH) { SearchWorkerResult(navigationActions) }
```

Comment:
This must surely be a mistake as we've decided that QuickFixFinderScreen was the one merging both the search and announcement processes.

Reply:
Of course this was just for test purposes